Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -122,6 +122,16 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
             this._fontVariationsGroup.rows = [emptyRow];
         }
+
+        console.log("In Font Details Panel");
+        console.log(this.nodeStyles.usedFonts);
+
+        for (let fontName of this.nodeStyles.usedFonts) {
+            let row = document.createElement("div");
+            row.textContent = fontName;
+            row.style.fontFamily = fontName;
```

Comment:
this likely wont work as expected because the frontend is an entirely separate page from the backend, meaning that the `@font-family`/etc. to load the font wont exist

can you attach a screen shot of what this would look like?

Reply:
My goal is to create a for loop to add each of the fontNames to each row and also display each font in the font style.

Reply:
I will try and move this inside of lines 232 to 237.